opendatahub-io/odh-dashboard
# Fix visual isses on Enabled/Explore tiles

Closes [RHOAIENG-16896](https://issues.redhat.com/browse/RHOAIENG-16896)

## Description
Fixes visuals for tiles when disabled. Updates the colors for the actions and labels so there is better contrast in dark mode. Fix alignment of Labels on Explore page tiles.

## How Has This Been Tested?
- Navigate to `Applications -> Enabled`
- View in both light and dark mode
- Verify the visuals

- Navigate to `Applications -> Explore`
- View in both light and dark mode
- Verify the visuals

## Test Impact
No impact on automated tests, purely a visual change

## Screen shots
![image](https://github.com/user-attachments/assets/8eb0a035-9c82-42e8-9d5a-97979ac45256)

![image](https://github.com/user-attachments/assets/577fe5a2-9715-42e2-b296-3f224cb8ab62)

![image](https://github.com/user-attachments/assets/0922b1ae-4605-4a25-a348-e994459bf169)

![image](https://github.com/user-attachments/assets/8f1c99c1-157b-49a8-9dff-567864ca94c7)

## Request review criteria:



Self checklist (all need to be checked):

If you have UI changes: 


After the PR is posted & before it merges:
